### ISSUE TITLE
loading project and when I click inspect long loaded and WebGL has crashed

### ISSUE THREAD

**Issue by ViktorSh25** [avocode/avocode#1881]
IMPORTANT: Please try to describe how did the crash happen.

App v. `2.19.2`

**Comment by vojtatranta** [comment]
hi @ViktorSh25 ,
can you please tell me what the size of your RAM?

**Comment by KarelPetruzela** [comment]
Since last(or might be last two) version I am experiencing same issue but it occurs after some time working with avocode. And it even might be connected(in my case) with action export to svg but this I cannot say for sure. I have got 16GB of RAM, OS Linux, version 2.19.2.

---
* **Version**: 2.19.2
* **Web/Desktop**: Desktop
* **Platform**: linux
* **Avocode UID**: 64146
---
```
No stack trace available
```

**Comment by jakubzitny** [comment]
So what happens exactly and how often @KarelPetruzela, please?

Any more details about your issues @ViktorSh25?

**Comment by KarelPetruzela** [comment]
This happen when I try to export an icon to svg..
![image](https://user-images.githubusercontent.com/6229456/36723583-fbe5384e-1bb0-11e8-8c07-5a876627313b.png)

Its happening on this(https://app.avocode.com/view/e5dcbd1a553a44f6a86da60ae00dc6d9) sketch design. Specifically when I try to right click the check icon in the section "Sluzby a cenik". (then it just freeze for a moment) and then when I try to click on export to svg it freeze again and says the message shown above. But in the browser version of avocode it works.

**Comment by jakubzitny** [comment]
Yeah, the graphics engine in the desktop app and in browsers is a little bit different and it might be causing the crashes in the desktop app. Once it crashes we cannot really work with it and rendering/exporting assets is limited. We're trying to figure out better options for such cases but it will take some time until we figure out the ultimate solution.

For now I suggest you use the web app. And I'll leave this issue open so we keep it in mind that these cases are happening..
